### PR TITLE
test(mojaloop/#3819): prepare handler testing

### PR DESCRIPTION
**Pull request by kleyow**

### CODE REVIEW

**Review by vijayg10 — APPROVED** [review]

File: test/integration/handlers/transfers/handlers.test.js [hunk 1]
```diff
@@ -319,6 +319,151 @@ const prepareTestData = async (dataObj) => {
...
+  now: new Date(),
+  expiration: new Date((new Date()).getTime() + (24 * 60 * 60 * 1000)) // tomorrow
+}
+
+const prepareFxTestData = async (dataObj) => {
+  try {
+    const payer = await ParticipantHelper.prepareData(dataObj.payer.name, dataObj.sourceAmount.currency)
+    const fxp = await ParticipantHelper.prepareData(dataObj.fxp.name, dataObj.sourceAmount.currency)
+
+    const kafkacat = 'GROUP=abc; T=topic; TR=transfer; kafkacat -b localhost -G $GROUP $T-$TR-prepare $T-$TR-position $T-$TR-fulfil $T-$TR-get $T-admin-$TR $T-notification-event $T-bulk-prepare'
```
Comment (geka-evk) [inline test/integration/handlers/transfers/handlers.test.js:352]: What for we need this `kafkacat`?

File: test/integration/handlers/transfers/handlers.test.js [hunk 2]
```diff
@@ -319,6 +319,151 @@ const prepareTestData = async (dataObj) => {
...
+      targetAmount: {
+        currency: dataObj.targetAmount.currency,
+        amount: dataObj.targetAmount.amount
+      }
+    }
+
+    const fxPrepareHeaders = {
+      'fspiop-source': payer.participant.name,
+      'fspiop-destination': fxp.participant.name,
+      'content-type': 'application/vnd.interoperability.transfers+json;version=1.1'
```
Comment (geka-evk) [inline test/integration/handlers/transfers/handlers.test.js:410]: maybe, the content-type should be `application/vnd.interoperability.**fx**Transfers+json;version=1.1` ?

File: test/integration/handlers/transfers/handlers.test.js (hunk 1 above)
Comment (kleyow) [reply test/integration/handlers/transfers/handlers.test.js:352]: We don't, thats copy pasta from me.

File: test/integration/handlers/transfers/handlers.test.js (hunk 2 above)
Comment (kleyow) [reply test/integration/handlers/transfers/handlers.test.js]: done

**Review by geka-evk — APPROVED** [review]

**Pull request merged**